src/fidesops/ops/schemas/email/email.py

```diff
@@ -43,6 +44,20 @@ def get_verification_code_ttl_minutes(self) -> int:
         return self.verification_code_ttl_seconds // 60
 
 
+class FidesopsEmail(
+    BaseModel,
+    smart_union=True,
+    arbitrary_types_allowed=True,
+):
+    """A mapping of action_type to body_params"""
+
+    action_type: EmailActionType
+    body_params: Union[
+        SubjectIdentityVerificationBodyParams,
+        List[CheckpointActionRequired],
+    ]
```

Comment:
Only using this for SUBJECT_IDENTITY_VERIFICATION emails right now, just noting a location where we'll add to the body_params -